explainaboard/processors/processor.py

```diff
@@ -194,12 +194,16 @@ def _gen_external_stats(self, sys_info: SysOutputInfo) -> Any:
                         output_data=None,
                     )
                     dataset = loader.load()
-                except ValueError as e:
+                except FileNotFoundError as e:
```

Comment:
This is weird: `FileNotFoundError` is corresponding to the `ENOENT` system errno by design and using it for any other purpose confuses the entire system.

Reply:
Hi, @odashi.  Here `FileNotFoundError` is used to capture the error raised by DataLab when invalid dataset names are specified by users.  DataLab adopt huggingface dataset's strategy for dataset loading process, specifically,
* it will first download all datasets' dataloader script files [here]FileNotFoundError(https://github.com/ExpressAI/DataLab/tree/main/datasets)
* then judge if the user-specified dataset names could be found from the above folder,
* if not, `FileNotFoundError` will be raised.

Reply:
@pfliu-nlp I meant you need to revise the strategy in Datalab. It is dangerous to use built-in errors for other meanings.